docs/guides/canonical.rst

```diff
@@ -31,6 +31,16 @@ Check your :guilabel:`Admin` > :guilabel:`Domains` page for the domains that we
 Implementation
 --------------
 
+If you are using :doc:`Sphinx </intro/getting-started-with-sphinx>`,
+Read the Docs will set the value of the html_baseurl_ setting (if isn't already set) to your canonical domain.
+
+.. _html_baseurl: https://www.sphinx-doc.org/page/usage/configuration.html#confval-html_baseurl
+
+If you are using :doc:`MkDocs </intro/getting-started-with-mkdocs>`,
+you can use the site_url_ setting.
```

Comment:
Do we not automatically support this at all for mkdocs?

Reply:
No, but we can do it if we want https://github.com/readthedocs/readthedocs.org/issues/4820 :)